js/output/sql/sql-output.js

```diff
@@ -65,95 +65,97 @@ window.SQLOutput = Backbone.View.extend({
         statement = statement || "";
         statement = statement.toUpperCase();
 
+        // Error messages take form: 'near \"%T\": syntax error'
```

Comment:
I grepped the SQLite C source, it looks like they all take this form unless it's in fulltextsearch mode, and sqlite.js doesn't seem to enable that by default (https://github.com/kripken/sql.js/pull/199)